static/js/event-handlers.js

```diff
@@ -611,29 +611,48 @@ function logOptionSingleHandler() {
     $('#logs-result-container').removeClass('multi');
     $('#views-container .btn-group .btn').removeClass('active');
     $('#log-opt-single-btn').addClass('active');
+    
     logsColumnDefs.forEach(function (colDef, index) {
-        if (colDef.field === "logs"){
+        if (colDef.field === "logs") {
             colDef.cellStyle = null;
             colDef.autoHeight = null;
             colDef.cellRenderer = function(params) {
                 const data = params.data || {};
                 let logString = '';
                 let addSeparator = false;
+                
+                // Handle 'event' field specifically
+                if (data.event && Array.isArray(data.event)) {
```

Comment:
Can we have a generic solution instead of specifying event as the column name, considering the column name could be anything?